lib/graph/Collaboratory/Portal.js

```diff
@@ -130,3 +130,83 @@ COL_Portal.prototype.post = function (resource, data, callback) {
             callback(null, data, res);
         }));
 };
+
+
+/**
+ * Retrieves a list courses within a portal
+ *
+ * @param {Object} query
+ * @param {Function} callback
+ */
+COL_Portal.prototype.courses = function (query, callback) {
+    if (typeof query === "function") {
+        callback = query;
+        query = null;
+    }
+
+    return this.related('courses', query, callback);
+};
+
+
+/**
+ * Get collaborators associated with a given class
+ *
+ * @param {String} *id
+ * @param {Object} query
+ * @param {Function} callback
+ */
+COL_Portal.prototype.coursesCollaborators = function(id, query, callback) {
```

Comment:
Why not call it courseCollaborators instead of coursesCollaborators?

Reply:
typo :)